backend/config/mongoid.yml

```diff
@@ -3,17 +3,26 @@ development:
   clients:
     # Defines the default client. (required)
     default:
-      # Defines the name of the default database that Mongoid can connect to.
+      # Mongoid can connect to a URI accepted by the driver:
+      # uri: mongodb://user:password@mongodb.domain.com:27017/flaredown_development
+
+      # Otherwise define the parameters separately.
+      # This defines the name of the default database that Mongoid can connect to.
```

Comment:
Changes in this file are all comments. They are all from running `rails g mongoid:config` after upgrading Mongoid.